dlk/python/dlk/templates/manual/consts/input.tpl.cpp

```diff
@@ -25,13 +25,13 @@ limitations under the License.
 {% if node.transposed_data %}
 
 #if defined(RUN_ON_FPGA)
-{{ node.dtype.cpptype() }} {{ node.name }}[] = {
+Base<{{ node.dtype.cpptype() }}>::type {{ node.name }}_raw[] = {
```

Comment:
How about putting `static` here, to hide implementation? :thinking:

Reply:
Add `static`.